tests/parser/EmbeddedPython.cpp

```diff
@@ -180,7 +180,7 @@ BOOST_AUTO_TEST_CASE(Python_Constructor) {
     BOOST_CHECK(python_on.enabled());
 
     //.enabled() Can only have one Python interpreter active at any time
-    BOOST_CHECK_THROW(Python(Python::Enable::ON), std::logic_error);
+    BOOST_CHECK_THROW(Python python_throw(Python::Enable::ON), std::logic_error);
```

Comment:
That is way above my payscale. Care to explain what it does?

Reply:
Works around a bug in clang basically. It fails to parse the non-named instantation of the class. This just gives it a name, and that fixes the peoblem.

Reply:
basically, instead of
```
try {
  Python(Python::Enable::ON);
} catch (std::logic_error&) {
  return true;
}
return false;
```
it now reads
```
try {
  Python python_throw(Python::Enable::ON);
} catch (std::logic_error&) {
  return true;
}
return false;
```

Reply:
Aah, now I get it. `python_throw` is a variable name and not a function. :flushed: